src/main/java/org/mockbukkit/mockbukkit/block/data/LeavesDataMock.java

```diff
@@ -0,0 +1,84 @@
+package org.mockbukkit.mockbukkit.block.data;
+
+import com.google.common.base.Preconditions;
+import org.bukkit.Material;
+import org.bukkit.block.data.type.Leaves;
+import org.jetbrains.annotations.NotNull;
+
+import static org.mockbukkit.mockbukkit.block.data.BlockDataKey.DISTANCE;
+import static org.mockbukkit.mockbukkit.block.data.BlockDataKey.PERSISTENT;
+
+public class LeavesDataMock extends BlockDataMock implements Leaves
+{
+
+	private static final int MAX_DISTANCE = 7;
+	private static final int MIN_DISTANCE = 1;
```

Comment:
This values can be mined using MetaMiner. For example see the `TurleEgg` in `MaterialDataGenerator` :
https://github.com/MockBukkit/MockBukkit/blob/af34b1d30cbc32ff2055b07a72e379fe74aec96f/metaminer/src/main/java/org/mockbukkit/metaminer/internal/MaterialDataGenerator.java#L153-L157